subsys/usb/device_next/class/usbd_hid.c

```diff
@@ -231,19 +231,21 @@ static int handle_get_report(const struct device *dev,
 	const uint8_t id = HID_GET_REPORT_ID(setup->wValue);
 	struct hid_device_data *const ddata = dev->data;
 	const struct hid_device_ops *ops = ddata->ops;
+	size_t size = net_buf_tailroom(buf);
+	uint8_t *data = net_buf_add(buf, size);
```

Comment:
That is not correct fix, we need to change a bit more, please open a bug report.

Reply:
Opened a bug report: https://github.com/zephyrproject-rtos/zephyr/issues/73457
Could you please let me know when we can expect proper fix? Can we use this commit as a temporary workaround locally (would it have side-effects)?